bokjido/src/main/java/com/projectbusan/bokjido/service/WelfareFacility.java

```diff
@@ -29,6 +42,42 @@ public void create(FacilityDTO.FacilityDto facilityDto) {
     @Transactional
     public Optional<Facility> searchById(Long id) { return facilityRepository.findById(id); }
 
+    // <<-- 복지 건물 조회 By 주소 -->>
+    public List<Facility> searchByLocation(String searchTerm) {
+        return facilityRepository.findFacilitiesByLocationContaining(searchTerm);
+    }
+
     // <<-- 전체 복지 건물 조회 -->>
     public List<Facility> loadAll() { return facilityRepository.findAll(); }
+
+    // <<-- 복지 시설 리뷰 생성 -->>
+    public void createReview(User user, FacilityReviewDTO.FacilityReviewRequestDTO facilityReviewRequestDTO) {
+        FacilityReview facilityReview = FacilityReview.builder()
+                .user(user)
+                .facility(findFacility(facilityReviewRequestDTO.getFacilityId()))
+                .benefit(findBenefits(facilityReviewRequestDTO.getBenefitId()))
+                .clean(facilityReviewRequestDTO.getClean())
+                .kindness(facilityReviewRequestDTO.getKindness())
+                .parking(facilityReviewRequestDTO.getParking())
+                .content(facilityReviewRequestDTO.getContent())
+                .build();
+
+        facilityReviewRepository.save(facilityReview);
+
+    }
+
+    // <<-- 복지 시설 리뷰 조회 -->>
+    public List<FacilityReview> getReview(Long facilityId) {
+        return facilityReviewRepository.findByFacilityId(facilityId);
+    }
+
+    private Benefit findBenefits(Long id){
+        return benefitsRepository.findById(id).orElseThrow(()
+                -> new CustomException(ErrorCode.BENEFITS_NOT_FOUND_ERROR, "해당 복지 서비스는 존재하지 않습니다."));
+    }
+
+    private Facility findFacility(Long id){
+        return facilityRepository.findById(id).orElseThrow(()
+                -> new CustomException(ErrorCode.BENEFITS_NOT_FOUND_ERROR, "해당 복지 시설은 존재하지 않습니다."));
+    }
```

Comment:
앗 이부분 같은 오류를 return하는데 저도 이 부분 수정을 안했네요.. 😅
ErrorCode에 추가하고 아래와 같이 수정해야 할 것 같습니다 ㅎㅎ
```suggestion
    private Benefit findBenefits(Long id){
        return benefitsRepository.findById(id).orElseThrow(()
                -> new CustomException(ErrorCode.BENEFITS_NOT_FOUND_ERROR, "해당 복지 서비스는 존재하지 않습니다."));
    }

    private Facility findFacility(Long id){
        return facilityRepository.findById(id).orElseThrow(()
                -> new CustomException(ErrorCode.FACILITY_NOT_FOUND_ERROR, "해당 복지 시설은 존재하지 않습니다."));
    }
```

Reply:
앗...저도 미처 확인하지 못했네요 수정 완료했습니다 :D